airbyte-integrations/connectors/destination-databricks/.gitignore

```diff
@@ -0,0 +1 @@
+lib/SparkJDBC42.jar
```

Comment:
```suggestion
# Get this from https://databricks.com/spark/jdbc-drivers-download
# this is pedantic but necessary for legal reasons. By downloading from the UI you are agreeing to the T&Cs
lib/SparkJDBC42.jar
```

Reply:
Should we explicitly tell people how to build this connector now? I am deliberately not including the URL hoping that everyone is too lazy to do that until we figure out how to publish a the public version.

Reply:
well if they download it from there then they presumably accepted the T&C

Reply:
Ok.